src/utils.jl

```diff
@@ -92,7 +92,7 @@ function sideof(point::Point{3,T}, mesh::Mesh{3,T}) where {T}
   (eltype(mesh) <: Triangle) || return sideof(point, simplexify(mesh))
 
   z = last.(coordinates.(extrema(mesh)))
-  r = Ray(point, Vec(zero(T), zero(T), 2 * (z[2] - z[1])))
+  r = Ray(point, Vec{3,T}(zero(T), zero(T), 2 * (z[2] - z[1])))
```

Comment:
@eliascarv this one here is necessary? I thought that Julia would be able to infer the types from the other expressions.

Reply:
True, it was an unnecessary change.